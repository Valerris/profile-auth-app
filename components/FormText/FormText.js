import styles from "./FormText.module.css";

export default function FormText({ children }) {
	return <span className={styles.formText}>{children}</span>;
}
