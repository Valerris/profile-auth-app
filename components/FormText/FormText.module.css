.formText {
	display: block;
	line-height: 1.25rem;
	margin: 0 0 2.25rem;
	color: inherit;
}

.formText:last-child {
	margin-bottom: 0;
}
