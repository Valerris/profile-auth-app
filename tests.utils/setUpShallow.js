export default function (C, props) {
	return shallow(<C {...props} />);
}
