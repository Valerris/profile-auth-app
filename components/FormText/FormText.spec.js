import setUpShallow from "tests.utils/setUpShallow";
import FormText from "./FormText";

describe("<FormText />", () => {
	let cmp = null;
	beforeEach(() => {
		cmp = setUpShallow(FormText);
	});

	it("should render FormText component with props", () => {
		cmp = setUpShallow(FormText, { children: "Form Text" });
		expect(cmp).toMatchSnapshot();
	});

	it("should render FormText component without props", () => {
		cmp = setUpShallow(FormText);
		expect(cmp).toMatchSnapshot();
	});
});
