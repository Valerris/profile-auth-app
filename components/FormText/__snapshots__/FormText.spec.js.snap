// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormText /> should render FormText component with props 1`] = `
<span
  className="formText"
>
  Form Text
</span>
`;

exports[`<FormText /> should render FormText component without props 1`] = `
<span
  className="formText"
/>
`;
